unix_integration/resolver/src/unix_config.rs

```diff
@@ -81,29 +137,31 @@ pub struct KanidmUnixdConfig {
     pub hsm_type: HsmType,
     pub hsm_pin_path: String,
     pub tpm_tcti_name: String,
-    pub allow_local_account_override: Vec<String>,
+
+    pub kanidm_config: Option<KanidmConfig>,
+}
+
+#[derive(Debug)]
+pub struct KanidmConfig {
+    pub conn_timeout: u64,
+    pub request_timeout: u64,
+    pub pam_allowed_login_groups: Vec<String>,
+    pub extend: Vec<GroupMap>,
 }
 
-impl Default for KanidmUnixdConfig {
+impl Default for UnixdConfig {
     fn default() -> Self {
-        KanidmUnixdConfig::new()
+        UnixdConfig::new()
     }
 }
 
-impl Display for KanidmUnixdConfig {
+impl Display for UnixdConfig {
     fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
-        writeln!(f, "db_path: {}", &self.db_path)?;
+        writeln!(f, "cache_db_path: {}", &self.cache_db_path)?;
```

Comment:
Did we also want to write the persistent db path?

Reply:
Not yet, that's coming in a follow up PR. There is a longer/bigger plan here that I probably need to share with you.